.github/workflows/e2e_nightly_upgrade.yml

```
@@ -49,7 +49,7 @@ jobs:
     # Testing upgrade from '1.7.4.4', '1.7.5.2', '1.7.6.9', '1.7.7.1' to the last '1.7.7.x' and '1.7.8.x'(develop)
     strategy:
       matrix:
-        ps_start_version: ['1.7.4.4', '1.7.5.2', '1.7.6.9', '1.7.7.1']
+        ps_start_version: ['1.7.4.4', '1.7.5.2', '1.7.6.9', '1.7.7.8']
```

```suggestion
        ps_start_version: ['1.7.4.4', '1.7.5.2', '1.7.6.9', '1.7']
```
If it's based on the Docker Hub images, we should be able to use 1.7 to retrieve the latest available one

---

It's used also for the test library to get the files used.